lib/node_modules/@stdlib/math/base/complex/divide/docs/repl.txt

```diff
@@ -0,0 +1,31 @@
+
+{{alias}}( re1, im1, re2, im2 )
+    Divides two complex numbers.
+
+    Parameters
+    ----------
+    re1: number
+        Real component.
+
+    im1: number
+        Imaginary component.
+
+    re2: number
+        Real component.
+
+    im2: number
+        Imaginary component.
+
+    Returns
+    -------
+    out: Array
```

Comment:
Once updated, the type should be `Array|TypedArray|Object`. The latter values due to user's having the ability to provide such values as output arguments.